examples/e12_global_data/src/main.rs

```diff
@@ -89,10 +89,18 @@ impl EventHandler for Handler {
                 data_read.get::<MessageCount>().expect("Expected MessageCount in TypeMap.").clone()
             };
 
+            // To check how many "owo" there are in msg.content, we split the string with the "owo" pattern 
+            // and reduce the count of iterator returned by the split function by one.
+            let owo_in_msg = msg.content.to_lowercase().split("owo").count() - 1;
```

Comment:
Technical nit: to_ascii_lowercase(). Is more efficient and cannot cause false positives if some weird Unicode character has a lowercase mapping that ends with "o"

Reply:
Just noticed to_lowercase() is already run above. The lowercased text could be saved to avoid computing twice